From f8627a65b078d06034985c950c37eac8c518e875 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman
Date: Sat, 27 Jun 2020 04:19:42 +0200
Subject: arm64: dts: sun50i-pinephone: dldo4 must not be >= 1.8V

Some outputs from the RTL8723CS are connected to the PL port, which
is at 1.8V. This causes backlight consistency issues.

Signed-off-by: Ondrej Jirman <megous@megous.com>
---
 arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
index f3b6bbec93f7..19ee00d8ff68 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
@@ -688,7 +688,7 @@
 
 &reg_dldo4 {
 	regulator-min-microvolt = <1800000>;
-	regulator-max-microvolt = <3300000>;
+	regulator-max-microvolt = <1800000>;
 	regulator-name = "vcc-wifi-io";
 };
 
-- 
cgit v1.2.3

